TableViewKit/TableViewKitDataSource.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+open class TableViewKitDataSource: NSObject, UITableViewDataSource {
+
+    open unowned var manager: TableViewManager
+    open var sections: ObservableArray<Section> { return manager.sections }
```

Comment:
The idea is that if you want to customize the delegates without the need of forking the library you can subclass TableViewKitDataSource and access the manager.